packages/mdc-select/README.md

```diff
@@ -462,6 +463,8 @@ If you are using a JavaScript framework, such as React or Angular, you can creat
 | `handleFocus() => void` | Handles a focus event on the `select` element. |
 | `handleBlur() => void` | Handles a blur event on the `select` element. |
 | `handleClick(normalizedX: number) => void` | Sets the line ripple center to the normalizedX for the line ripple. |
+| `handleMenuOpened() => void` | Handles menu or menu surface opened event.
```

Comment:
Don't think you need to mention menu surface:
```
Handles menu opened event.
```

same with below.

Reply:
I think this depends on framework where the parent component encapsulates the child component's events. I want give option here that this callback can be bound to menu or menu surface event. 

In vanilla implementation we listen to Menu Surface opened event.